meta/store.go

```diff
@@ -1039,6 +1039,29 @@ func (s *Store) CreateDatabase(name string) (*DatabaseInfo, error) {
 	return s.Database(name)
 }
 
+// CreateDatabaseWithRetentionPolicy creates a new database with an explicit retention policy in the store.
+func (s *Store) CreateDatabaseWithRetentionPolicy(name string, rpi *RetentionPolicyInfo) (*DatabaseInfo, error) {
+	if err := s.exec(internal.Command_CreateDatabaseCommand, internal.E_CreateDatabaseCommand_Command,
```

Comment:
No, I think it is fine. Perhaps a comment making it clear. Your call.

On Saturday, November 28, 2015, Paulo Pires <notifications@github.com>
wrote:

> In meta/store.go
> <https://github.com/influxdb/influxdb/pull/4866#discussion_r46081297>:
>
> > @@ -1039,6 +1039,29 @@ func (s *Store) CreateDatabase(name string) (*DatabaseInfo, error) {
> >  	return s.Database(name)
> >  }
> >
> > +// CreateDatabaseWithRetentionPolicy creates a new database with an explicit retention policy in the store.
> > +func (s *Store) CreateDatabaseWithRetentionPolicy(name string, rpi *RetentionPolicyInfo) (*DatabaseInfo, error) {
> > +	if err := s.exec(internal.Command_CreateDatabaseCommand, internal.E_CreateDatabaseCommand_Command,
>
> It is deliberate. Should this be different?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/influxdb/influxdb/pull/4866/files#r46081297>.
>